lib/bambora/adapters/query_string_response.rb

```diff
@@ -8,7 +8,7 @@ def to_h
       parsed_response = super
       return error_response if parsed_response.values.flatten.empty? # We didn't get a query string back.
 
-      parsed_response.each_with_object({}) { |(key, val), obj| obj[key] = val.length == 1 ? val.first : val }
+      parsed_response.transform_values { |val| val.length == 1 ? val.first : val }
```

Comment:
Linting changes from the Rubocop change moving us to 3.0 compatibility. These were all from their "auto-correct" package.